packages/flex-dev-utils/src/packages/lib/getLatestFlexUIVersion.ts

```diff
@@ -0,0 +1,23 @@
+import semver from 'semver';
+
+import getRegistryVersion from './getRegistryVersion';
+
+/**
+ * Returns the latest flex ui version for a given major
+ * Will search through latest, beta, and alpha versions for a matching verion (in that order)
+ * @param version the flex ui major version
+ */
+export default async function getLatestFlexUIVersion(majorVersion: 1 | 2 | 3 | 4): Promise<string> {
+  const tags: ('latest' | 'beta' | 'alpha')[] = ['latest', 'beta', 'alpha'];
```

Comment:
at this point, might as well create a type in the `getRegistryVersion` file and export it and use it here

```ts
// getRegistryVersion.ts
export type Tag = 'latest' | 'beta' | 'alpha';
```

and use that here